core/src/main/scala/org/ensime/config/EnsimeConfigProtocol.scala

```diff
@@ -29,22 +27,14 @@ object EnsimeConfigProtocol {
 
   def parse(config: String): EnsimeConfig = {
     val raw = config.parseSexp.convertTo[EnsimeConfig]
-    validated(raw).copy(javaLibs = inferJavaLibs(raw.javaHome))
+    validated(raw)
   }
 
-  // there are lots of JRE libs, but most people only care about
-  // rt.jar --- this could be parameterised.
-  private def inferJavaLibs(javaHome: File): List[File] =
-    // WORKAROUND https://github.com/ensime/ensime-server/issues/886
-    // speeds up the emacs integration tests significantly,
-    if (Properties.envOrNone("ENSIME_SKIP_JRE_INDEX").isDefined) Nil
-    else javaHome.tree.filter(_.getName == "rt.jar").toList
```

Comment:
I stand corrected, the `rt.jar` *is* the only jar we care about... but the logic should live here, not in the `case class`